extensions/linux-source-package.sh

```
@@ -0,0 +1,60 @@
+}
+
+function add_host_dependencies__add_fakeroot() {
+	display_alert "Adding packages to host dependencies" "${EXTENSION}" "info"
+	EXTRA_BUILD_DEPS="${EXTRA_BUILD_DEPS} fakeroot"
+}
+
+function armbian_kernel_config__create_ksrc_package() {
+	if [[ -f .config ]]; then
+
+		#( set -o posix ; set )
+		echo ${kernel_version_family}
```

Shellcheck (0.10.0)

[warning] kernel_version_family is referenced but not assigned.

(SC2154)